amacc.c

```diff
@@ -1231,6 +1250,23 @@ int *codegen(int *jitmem, int *jitmap)
         case MUL:
             *je++ = 0xe49d1004; *je++ = 0xe0000091; // pop {r1}; mul r0, r1, r0
             break;
+        case DIV:
+        case MOD:
+            *je++ = 0xe52d0004;                     // push {r0}
+            if (elf) {
+                tmp = (int) plt_func_addr[i - OPEN];
+            } else {
+                void *handle = dlopen("libgcc_s.so.1", 1);
+                if (!handle) fatal("libgcc_s.so.1 open error!");
+                tmp = (int) dlsym(handle, scnames[i - OPEN]);
+            }
+            *je++ = 0xe49d0004 | (1 << 12); // pop r1
+            *je++ = 0xe49d0004 | (0 << 12); // pop r0
+            *je++ = 0xe28fe000;                          // add lr, pc, #0
+            if (!imm0) imm0 = je;
+            *il++ = (int) je++ + 1;
+            *iv++ = tmp;
+            break;
```

Comment:
No need to add these lines for DIV and MOD.
`default:` case shall handle all of the system calls.
It obviously you copy the code from `default:`, this should be fixed.